frame/support/src/storage/generator/nmap.rs

```diff
@@ -538,22 +507,18 @@ mod test_iterators {
 	#[test]
 	fn n_map_double_map_identical_key() {
 		sp_io::TestExternalities::default().execute_with(|| {
+			use crate::hash::{Blake2_128Concat, Twox64Concat};
+
+			type NMap = self::frame_system::NMap<Runtime>;
```

Comment:
is this correct? we are creating a type alias and then shadowing it later? why?

Reply:
If we don't create a type alias, we need to specify generics in the following code, I want to keep changes to the minimum.